logrotate.c

```diff
@@ -106,6 +106,16 @@ unsigned int hashSize;
 int numLogs = 0;
 int debug = 0;
 char *mailCommand = DEFAULT_MAIL_COMMAND;
+/* This is the declaration. Note the "2" is needed in the definition (logrotate.h), too! */
+const char * compress_cmd_list[][2] = {
```

Comment:
Why do you define the array in a different module than the one that actually uses it?  Please make it a static array in the appropriate module to limit visibility of the symbol.